pecan-waffle.psm1

```diff
@@ -1101,6 +1101,31 @@ function New-PWItem{
 }
 Set-Alias Add-Item New-PWItem -Description 'obsolete: This was added for back compat and will be removed soon'
 
+function Remove-PWTemplate{   
+    [cmdletbinding()]
+    param(
+        [Parameter(Position=1,Mandatory=$true)]
+        [ValidateNotNullOrEmpty()]
```

Comment:
I think it would be better to remove this validation and then just skip empty values in `TemplateNames`. This works better for the pipeline as well.